storybookjs/storybook
# CLI/HTML: Add interactions to cli template

Issue: #16722

## What I did

Much like #17345, I added addon interactions to the HTML template in the CLI

## How to test

Build `cli` locally and run `sb repro` to select html as a framework, or run `sb init`


If your answer is yes to any of these, please make sure to include it in your PR.
